__tests__/supertest.test.js

```diff
@@ -299,5 +297,41 @@ describe('GET /api/users', () => {
         });
         expect(users.length).toBe(4);
       });
-  })
+  });
+});
+
+describe('GET /api/reviews queries', () => {
+  it("should return an array of review objects which should conatain all of the following properties: review_id, title, designer, owner, review_img_url, category, created_at, votes, comment_count in ascending order, sorted_by review_id and only showing teh category of social deduction", () => {
+    return request(app)
+      .get("/api/reviews?category=social%20deduction&sort_by=review_id&order=asc")
+      .expect(200)
+      .then((res) => {
+        const reviews = res.body.reviews;
+        reviews.forEach((review) => {
+          expect(review).toMatchObject({
+            review_id: expect.any(Number),
+            title: expect.any(String),
+            designer: expect.any(String),
+            owner: expect.any(String),
+            review_img_url: expect.any(String),
+            category: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(String),
+          });
+        });
+        expect(reviews).toBeSorted("review_id", {
+          ascending: true,
+        });
+        expect(reviews.length).toBe(11);
+      });
+  });
+  it("invalid category, order or sort_by, returns a 400", () => {
```

Comment:
I would separate out these two tests, currently if one is correctly failing then it will cover up the other that is incorrectly passing. 

I would also add a 200 test for a category query that is valid, but has no associated reviews, returning an empty array